src/main/java/org/mockito/internal/junit/ArgMismatchFinder.java

```diff
@@ -0,0 +1,28 @@
+package org.mockito.internal.junit;
+
+import org.mockito.internal.invocation.finder.AllInvocationsFinder;
+import org.mockito.invocation.Invocation;
+import org.mockito.internal.invocation.Stubbing;
+
+/**
+ * For given mocks, finds stubbing arg mismatches
+ */
+class ArgMismatchFinder {
```

Comment:
@szczepiq 
Why do we need an instance of `ArgMismatchFinder`? Is this class intended to be subclassed so `getStubbingArgMismatch(..)` can be overriden?